app/views/posts/index.html.erb

```diff
@@ -1,3 +1,20 @@
-<h1>Posts Page</h1>
+<div class="user-specific-post">
+  <%= render partial: "users/user_info", locals: {user: @user} %>
+
+  <div class="post-info">
+    <% @posts.each do |post| %>
+      <%= render partial: "users/user_post_card", locals: {post: post} %>
+
+      <div class="comments border">
+        <% post.most_recent_comments.each do |comment| %>
+          <p><%= comment.author.name %>: <%= comment.text %></p>
+        <% end %>
+      </div>
+    <% end %>
+  </div>
+</div>
```

Comment:
## [OPTIONAL]
- [ ] Kindly make sure that each post is a link so that when the user clicks on the post he should be led to the single post that will display all its comments(you have already that page post#show)

Reply:
Thanks @andersonlebon